### ISSUE TITLE
[Support]: Do I need a coral device?

### ISSUE THREAD

**Issue by BrunoTeixeira1996** [blakeblackshear/frigate#8216]
### Describe the problem you are having

I am planning to use Frigate on a tp-link webcam so I can watch my puppy while I am  not at home.
Frigate will run on a Debian VM inside Proxmox but I am not sure (after reading the documentation) If I need any coral device.
I am not interested in face or person recognition.

### Version

Nothing

### Frigate config file

```yaml
Nothing
```

### Relevant log output

```shell
Nothing
```

### FFprobe output from your camera

```shell
Nothing
```

### Frigate stats

```json
Nothing
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

CPU (no coral)

### Network connection

Wireless

### Camera make and model

tp-link

**Comment by NickM-27** [comment]
if you don't want any object detection, then a coral is not needed. A GPU would still be recommended because frames still need to be decoded

**Comment by BrunoTeixeira1996** [comment]
@NickM-27 I have a small GPU that its only used to access the server via HDMI cable. Not sure if that is enough? Maybe it will be a bit slow but I don't mind since I just want to make sure the puppy is safe and well

**Comment by NickM-27** [comment]
yeah that would be fine, decoding is a relatively minimal workload that most users use an integrated GPU for
